velox/type/UnscaledLongDecimal.h

```diff
@@ -161,6 +161,21 @@ struct UnscaledLongDecimal {
     return *this;
   }
 
+  static inline void serialize(
```

Comment:
put serializedData after longDecimal as it is function output?

Reply:
done